train_unet.py

```diff
@@ -0,0 +1,119 @@
+
+    def create_parser():
+        parser = argparse.ArgumentParser(description='Train UNet 3D.')
+        parser.add_argument('--dataset_json', type=str, required=True)
+        parser.add_argument('--division_json', type=str, required=True)
+        parser.add_argument('--log_dir', type=str, required=True)
+        parser.add_argument('--device', type=str, required=True)
+        parser.add_argument('--epochs', type=int, required=True)
+        parser.add_argument('--batch_size', type=int)
+        parser.add_argument('--patience', type=int, default=10)
+        parser.add_argument('--learning_rate', type=float, default=0.001)
+        parser.add_argument('--train_valid_ratio', type=float, default=0.8)
+        parser.add_argument('--input_size', type=int, default=240)
+        parser.add_argument('--progress_bar', type=int, default=0)
+
+        return parser
+
+
+    parser = create_parser()
+    args = parser.parse_args()
+
+    volumes_transformations = trfs.Compose([transformations.NiftiToTorchDimensionsReorderTransformation(),
+                                            trfs.Lambda(lambda x: x[3, :, :, :]),
+                                            trfs.Lambda(lambda x: np.expand_dims(x, 0)),
+                                            trfs.Lambda(lambda x: torch.from_numpy(x)),
+                                            trfs.Lambda(
+                                                lambda x: F.pad(x, [0, 0, 0, 0, 5, 0]) if x.shape[1] % 2 != 0 else x),
+                                            trfs.Lambda(lambda x: F.interpolate(x, size=args.input_size)),
+                                            transformations.StandardizeVolume(),
+                                            trfs.Lambda(lambda x: x.float()),
+                                            ])
+    masks_transformations = trfs.Compose([trfs.Lambda(lambda x: np.expand_dims(x, 3)),
+                                          transformations.NiftiToTorchDimensionsReorderTransformation(),
+                                          trfs.Lambda(lambda x: torch.from_numpy(x)),
+                                          transformations.OneHotEncoding([0, 1, 2, 3]),
+                                          trfs.Lambda(
+                                              lambda x: F.pad(x, [0, 0, 0, 0, 5, 0]) if x.shape[1] % 16 != 0 else x),
+                                          trfs.Lambda(lambda x: F.interpolate(x, size=args.input_size)),
+                                          trfs.Lambda(lambda x: x.float())
+                                          ])
+
+    volumes_paths, masks_paths = read_dataset_json(args.dataset_json)
+    volumes_set = datasets.NiftiFolder(volumes_paths, volumes_transformations)
+    masks_set = datasets.NiftiFolder(masks_paths, masks_transformations)
+    combined_set = datasets.CombinedDataset(volumes_set, masks_set)
+    with open(args.division_json, "r") as division_file:
+        indeces = json.load(division_file)
```

Comment:
typo in indeces

Reply:
Where is the typo?

Reply:
I guess you mean plural of `index`, so it's `indices`

Reply:
Changed